experiments/billable/src/app/pages/invoiceDetail/FetchInvoice.tsx

```diff
@@ -1,11 +1,8 @@
 "use server";
 
-import React from "react";
 import { getInvoice } from "../../services/invoices";
 
-export function FetchInvoice(props: { id: number, children: Function }) {
-
-  const invoice = React.use(getInvoice(props.id, 1));
+export async function FetchInvoice(props: { id: number, children: Function }) {
+  const invoice = await getInvoice(props.id, 1);
```

Comment:
While working on a fix for the original issue, I discovered that `use()` won't work either - for SSR renders, react gets the RSC version of use(), which it doesn't know how to use.

I'll work on a fix for this next (for use() and for more general usage of React APIs) so that SSR and RSC versions of react can co-exist better.